applications/crossbar/priv/api/swagger.json

```diff
@@ -12702,37 +12702,9 @@
         "version": "2.0"
     },
     "paths": {
-        "/v2/about": {
-            "get": {}
-        },
```

Comment:
there are more. Apparently all paths without masquerading part are not discovered but hard coded